reference/providers/kubernetes-v2/index.md

```diff
@@ -137,13 +137,17 @@ It is worth noting that the resource mapping exists primarily to render
 resources in the UI according to Spinnaker conventions. It does not affect how
 resources are deployed or managed.
 
-There are three major groupings of resources in Spinnaker, Server Groups, Load
-Balancers, and Security Groups. They correspond to Kubernetes resource kinds as
-follows:
+There are three major groupings of resources in Spinnaker:
 
-* Workloads ≈ Spinnaker Server Groups
-* Services, Ingresses ≈ Load Balancers
-* NetworkPolicies ≈ Security Groups
+* server groups
+* load balancers
+* security groups
+
+These correspond to Kubernetes resource kinds as follows:
+
+* server groups ≈ workloads
+* load balancers ≈ services, ingresses
+* security groups ≈ network policies
```

Comment:
NetworkPolicies is one word - I lean towards keeping these capitalized to refer to the proper kubernetes resource